libraries/transformer/src/main/java/androidx/media3/transformer/AudioSampleExporter.java

```diff
@@ -94,7 +94,7 @@ public AudioSampleExporter(
   }
 
   @Override
-  public AudioGraphInput getInput(EditedMediaItem editedMediaItem, Format format)
+  public AudioGraphInput getInput(EditedMediaItem editedMediaItem, Format format, int sequenceIndex)
```

Comment:
@claincly  not sure but does Audio also need this information?

Reply:
I'm not sure either, but I think it's OK since it's from the `SampleExporter` API change.